accelerator/core/core.h

```diff
@@ -107,7 +107,8 @@ status_t ta_send_transfer(const ta_config_t* const info, const iota_config_t* co
  * @param[in] info Tangle-accelerator configuration variables
  * @param[in] iconf IOTA API parameter configurations
  * @param[in] service IRI node end point service
- * @param[in] trytes Trytes that will be attached to tangle
+ * @param[in, out] trytes Trytes that will be attached to tangle. The output trytes are the ones with completed PoW and
+ * Tangle broadcasting. and broadcast to Tangle
```

Comment:
Rewrite the `with completed PoW and Tangle broadcasting. and broadcast to Tangle` description.